src/main/webapp/css/inter.css

```
@@ -0,0 +1,27 @@
+ * Inter is designed for computer screens with a tall x-height for readability
+ * at small sizes, and features tabular number support for data alignment.
+ *
+ * License: SIL Open Font License 1.1 (see font/Inter/LICENSE.txt)
+ * Source: https://github.com/rsms/inter
+ * Version: 4.1
+ *
+ * @since 2026-03-21
+ */
+
+@font-face {
+    font-family: 'Inter';
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)